src/pluginutils.c

```diff
@@ -52,6 +52,16 @@ typedef struct
 PluginDocDataProxy;
 
 
+/** Returns the runtime API version Geany was compiled with.
+ *
+ * @return Geany's API version
```

Comment:
@codebrainz indeed, looks nice :) (I knew we already mentioned something like that!  I wasn't crazy -- not about that, at least)